pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/OtherIdentifiersRule.scala

```diff
@@ -23,6 +23,8 @@ object OtherIdentifiersRule extends FieldMergeRule with MergerLogging {
 
   type FieldData = List[SourceIdentifier]
 
+  private val otherIdentifiersTypeAllowList = Set("wellcome-digcode")
```

Comment:
We should have a comment here that explains:

1. What this list is
2. Why each member of it exists